src/alfasim_score/constants.py

```diff
@@ -64,3 +63,6 @@
 # set default value for annulus for the plugin APB
 # there is no such option in the SCORE input so use this default value
 HAS_FLUID_RETURN = True
+
+# total number of walls in the output
+TOTAL_WALLS = 6
```

Comment:
Couldn't we have more than 6 walls?

Reply:
I think if we modify this number in the future we just need to make sure to syncronize the value here, and let the `ScoreOutputBuilder` filter only the actually generated values for `N` anulli (informed in the SCORE input file).

Reply:
I got it.
Anyway, when testing the more complex configurations we will know it, thus we can update plugin and converter.